### PR TITLE
Private Chat docs 

### PR DESCRIPTION
**Pull request by narodnik-intern**
just a rough draft , open for suggestions and edits

### CODE REVIEW

**Comment by narodnik-intern** [comment]
will resolve the CI checks ...

File: doc/src/misc/ircd/private_message.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+```toml
+[private_key.”your_private_key_goes_here”]
+```
+
+to share public key with a user you can use the command 
+
+     /query User_A  “Hi this is my  publickey: XXXXXX"
+
+Note : this messgae will be publically visible on the IRC chat anyone running the irc demon can view this in logs
```
Comment (parazyd) [inline doc/src/misc/ircd/private_message.md:26]: messgae -> message

Comment (parazyd) [reply doc/src/misc/ircd/private_message.md:26]: publically -> publicly

**Comment by parazyd** [comment]
Hey, thanks.

There's some grammar mistakes so I'd recommend to fix them.
Also the general approach to markdown files is to wrap them at 72 columns,
so please to that as well before we merge it to the book.

**Comment by narodnik-intern** [comment]
hey @parazyd  sorry for the delay, updated as per the request.
Although i still didn't follow the 72 columns rule for the logs example as it wasn't looking good , i hope that works!!

**Pull request merged**